docs/_ext/core.py

```diff
@@ -123,7 +123,7 @@ def media_photo(_file, _type):
         return '/_static/img/speakers/missing.jpg'
 
     def speaker_photo(file):
-        return media_photo(file, 'speaker')
```

Comment:
That took me a surprisingly long time to track down.

Reply:
Huh, I thought I had speaker pics rendering properly with previous speakers. Weird.

Reply:
Maybe we reverted somehow?